com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -177,6 +182,26 @@ internal set
         /// </summary>
         public bool AutoObjectParentSync = true;
 
+        public InterestSettings InterestSettingsOverride;
+        public InterestSettings InterestSettings
```

Comment:
From an API design perspective, you have redundant setters here. I think it would be better if you had a `GetInterestSettings()` method to replace `InterestSettings`, or alternatively a `SetInterestSettingsOverride` method and make `InterestSettingsOverride` private.